src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -344,6 +349,11 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
       }
     }
 
+  lazy val booleanLiteral: Parser[BooleanLiteral] = accept("BooleanLiteral", {
+    case lexical.Keyword("true") => BooleanLiteral(true)
+    case lexical.Keyword("false") => BooleanLiteral(false)
+  })
```

Comment:
Why not the simpler
```scala
  lazy val ... = (
      "true" ^^^ BooleanLiteral(true)
    | "false" ^^^ BooleanLiteral(false)
  )
```
?

Reply:
Because I didn't know that was an option. 🙂 Updated in 8e8219a.